search/controllers/api/__init__.py

```diff
@@ -1,30 +1,25 @@
 """Controller for search API requests."""
 
 from collections import defaultdict
-from datetime import date, datetime
-import re
-
 from typing import Tuple, Dict, Any, Optional, List, Union
 from mypy_extensions import TypedDict
 
-from dateutil.relativedelta import relativedelta
 import dateutil.parser
 import pytz
 from pytz import timezone
-from werkzeug.datastructures import MultiDict, ImmutableMultiDict
-from werkzeug.exceptions import InternalServerError, BadRequest, NotFound
-from flask import url_for
+from werkzeug.datastructures import MultiDict
+from werkzeug.exceptions import BadRequest, NotFound
 
 from arxiv import status, taxonomy
 from arxiv.base import logging
 
-from search.services import index, fulltext, metadata
+from search.services import index
 from search.controllers.util import paginate
-from ...domain import Query, APIQuery, FieldedSearchList, FieldedSearchTerm, \
-    DateRange, ClassificationList, Classification, asdict, DocumentSet, \
-    Document, ClassicAPIQuery
-from ...domain.api import Phrase, Term, Operator, Field
-from .classic_parser import parse_classic_query
+from search.domain import (
```

Comment:
👍 
We've been trying to move away from relative imports for clarity, so this is helpful.